projects/ngx-d3timeline/src/lib/store/state.ts

```diff
@@ -12,6 +13,7 @@ export interface State {
   lastDraggedActivityId: identifier;
   dragEvent: TimelineDragEvent;
   zoomEvent: any;
+  hoveredEvent: HoverEvent;
```

Comment:
```suggestion
  hoverEvent: HoverEvent;
```
is consistent with zoomEvent and dragEvent

Reply:
done